Xamarin.Forms.Core.UnitTests/ButtonUnitTest.cs

```diff
@@ -316,6 +316,14 @@ public void ButtonBorderRadiusSetMinusOne()
 			Assert.AreEqual(-1, button.BorderRadius);
 		}
 
+		[Test]
+		public void SetLetterSpacing()
```

Comment:
those tests aren't required. LetterSpacing is a BindableProperty, and I trust BindableProperties to trigger INPC with the right name (by trust, I mean it's tested elsewhere)